actions/smoke-test

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+sys.path.append('lib')
+
+from charmhelpers.core import hookenv
+from charmhelpers.core.host import chdir, chownr
+from charms import layer
+from charms.layer.apache_bigtop_base import Bigtop
+from charms.reactive import is_state
+from jujubigdata.utils import run_as
+
+if not is_state('bigtop.available'):
+    hookenv.action_fail('Bigtop is not yet ready')
+
+
+bigtop = Bigtop()
+cfg = layer.options('apache-bigtop-base')
+smoke_components = cfg.get('bigtop_smoketest_components')
```

Comment:
It would be more consistent with our other stuff to use dashes here rather than underscores, right?